src/Plough.WebApi.Core/ApiClient.fs

```diff
@@ -52,6 +55,11 @@ open Core
 
 // The members of this class must be inlined so Fable can get the generic info
 type ApiClient(get, post, getBinary : string -> Task<Result<byte [], string>>, postBinary: byte[] -> string -> Task<Result<string,string>>) =
+    member x.GetRaw = get
+    member x.PostRaw = post
+    member x.GetBinaryRaw = getBinary
+    member x.PostBinaryRaw = postBinary
```

Comment:
When using the `FABLE_COMPILER` constant the members become inlined and the compiler complains because the members access private methods (here the arguments passed to the constructor) so I had to make them public using properties. The drawback is these become visible, but we can try to find an alternative if necessary.